### PR TITLE
Discord improvements

### PR DESCRIPTION
**Pull request by RobotoLev**
A couple of things related to the Discord module of tf2autobot

### CODE REVIEW

**Comment by RobotoLev** [comment]
The code from the last commit was implemented more than a month at the Derpy's request. I just found it on another PC.

**Review by idinium96 — APPROVED** [review]
I am terribly sorry for the late response. Merging this now. Thank you very much!

File: src/classes/DiscordBot.ts [hunk 1]
```diff
@@ -128,9 +160,12 @@ export default class DiscordBot {
...
         this.admins.forEach(admin => {
-            this.client.users.createDM(admin.discordID).catch(err => {
-                log.error('Failed to fetch DM channel with admin:', err);
-            });
+            const adminUser = this.client.users.resolve(admin.discordID);
+            if (!adminUser.bot) {
+                this.client.users.createDM(adminUser).catch(err => {
+                    log.error('Failed to fetch DM channel with admin:', err);
+                });
+            }
```
Comment (idinium96) [inline src/classes/DiscordBot.ts:168]: I am still getting an error here, where the `adminUser` is always null.

Comment (idinium96) [reply src/classes/DiscordBot.ts:168]: (You can test this on the current `development` branch.

Comment (RobotoLev) [reply src/classes/DiscordBot.ts:168]: I have some ideas why is it happening. Will check in the next day or two.

Comment (idinium96) [reply src/classes/DiscordBot.ts:168]: Alright thank you.

**Pull request merged**